apps/tui/chat_app.py

```
@@ -379,63 +414,101 @@ def __init__(self, argv: list[str] | None = None) -> None:
+                    configured = True  # never block chat on this probe
+                llm_config = await load_llm_config(conn, "llm.chat", fallback_key="llm")
+            self.model_name = llm_config.get("model", "") if isinstance(llm_config, dict) else ""
             self.registry = create_default_registry(self.pool)
             agent_profile = await get_agent_profile_context(self.dsn)
-            self.system_prompt = await _build_system_prompt(agent_profile, self.registry)
-
             if isinstance(agent_profile, dict):
                 self.agent_name = agent_profile.get("name", "Hexis")
-
+                self.mood = agent_profile.get("mood", "") or ""
         except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)